back/api/admin.py

```diff
@@ -50,17 +50,23 @@ class IdentityInline(admin.StackedInline):
 
 
 class MetadataContactInline(admin.StackedInline):
+    raw_id_fields = ['contact_person']
     model = MetadataContact
     extra = 1
-    list_filter = ['contact_person']
 
 
 class MetadataAdmin(CustomModelAdmin):
     inlines = [MetadataContactInline]
+    raw_id_fields = ['modified_user']
     search_fields = ['name', 'id_name']
     list_display = ('id_name', 'name')
     readonly_fields = ('image_tag', 'legend_tag')
 
+    def get_form(self, request, obj=None, change=False, **kwargs):
+        form = super(MetadataAdmin, self).get_form(request, obj, change, **kwargs)
```

Comment:
I do not get this line... but if it is working...

Reply:
This overrides get_form() from django. first it calls super() so it stills runs the original get_form(), then it sets an initial value for a field, so the user don't need to set it himself. It's how we set a default value in admin forms.